compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolResolver.java

```diff
@@ -515,6 +519,40 @@ BOperatorSymbol createBuiltinMethodSymbol(BLangBuiltInMethod method, BType type,
         return new BOperatorSymbol(names.fromString(method.getName()), null, opType, null, opcode);
     }
 
+    BOperatorSymbol createTypeAssertionSymbol(BType type, BType retType) {
+        List<BType> paramTypes = Lists.of(type);
+        BInvokableType opType = new BInvokableType(paramTypes, retType, null);
+        return new BOperatorSymbol(Names.ASSERTION_OP, null, opType, null, InstructionCodes.TYPE_ASSERTION);
+    }
+
+    BSymbol getExplicitlySimpleBasicTypedExpressionSymbol(BType sourceType, BType targetType) {
```

Comment:
wouldn't it be better to say `getExplicitlyTypedExpressionSymbol`, since other types are also resolved here?

Reply:
Simple basic type was used in the method name, since we call this only if the target type is a simple basic type. 

But then again, since explicitly typed expressions are defined for simple basic types, we can just say `getExplicitlyTypedExpressionSymbol`.